<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>systempage</class>
 <widget class="QWidget" name="systempage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统管理平台</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/背景2.jpg);</string>
  </property>
  <widget class="QPushButton" name="set_config">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日志文件：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="config_1"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="file_name_input">
      <property name="placeholderText">
       <string>_.txt</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="read_file">
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>最低信誉积分需：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>ECU_1-MPU可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="config_4">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>可更新的ECU的ID：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>可信任服务器ip地址：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>报文最大超时时间：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="config_3">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>可信任升级包下发服务器ip地址：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="config_5">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="config_6">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="config_2"/>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="config_8">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ECU_2-MPU可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>ECU_1-MCU-APP_2可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="config_7">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QLineEdit" name="config_9">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QLineEdit" name="config_10">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>ECU_1可更新应用id：</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>ECU_2可更新应用id：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ECU_1-MCU-APP_1可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2">
     <widget class="QLineEdit" name="config_12">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>ECU_2-MCU-APP_1可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>ECU_2-MCU-APP_2可兼容版本：</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1" colspan="2">
     <widget class="QLineEdit" name="config_13">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLineEdit" name="config_11">
      <property name="placeholderText">
       <string>以&quot;/&quot;分隔</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="set_config_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>20</width>
     <height>471</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>21</y>
     <width>981</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>系统日志：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="dbview">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
